src/app/views/followerships/_followership.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'users/user', user: followership.follower %>
```

Comment:
we can't render followership.follower directly?
```suggestion
<%= render followership.follower %>
```

Reply:
done